proxy/Cargo.toml

```diff
@@ -58,3 +59,4 @@ libc = "0.2"
 [dev-dependencies]
 quickcheck = { version = "0.6", default-features = false }
 conduit-proxy-controller-grpc = { path = "./controller-grpc" , features = ["arbitrary"] }
+flate2 = "1.0.1"
```

Comment:
TIOLI: `{ default-features = false, features = ["rust_backend"] }`, if it works, seems better.